v23.1/performance-recipes.md

```diff
@@ -77,15 +84,109 @@ This section provides solutions for common performance issues in your applicatio
 
 ### Transaction contention
 
-[Transaction contention](performance-best-practices-overview.html#transaction-contention) occurs when transactions issued from multiple clients at the same time operate on the same data. This can cause transactions to wait on each other (like when many people try to check out with the same cashier at a store) and decrease performance.
+[Transaction contention](performance-best-practices-overview.html#transaction-contention) is a state of conflict that occurs when a [transaction](transactions.html) is unable to complete due to another concurrent or recent transaction attempting to write to the same data. When CockroachDB experiences transaction contention, it will [automatically attempt to retry the failed transaction](transactions.html#automatic-retries) without involving the client (i.e., silently). If the automatic retry is not possible or fails, a [*transaction retry error*](transaction-retry-error-reference.html) is emitted to the client, requiring the client application to [retry the transaction](transaction-retry-error-reference.html#client-side-retry-handling). 
```

Comment:
Same comment as above. Perhaps clarify by describing that there are multiple problematic _symptoms_ of transaction contention.